lib/hammer_cli_foreman/ssh_keys.rb

```diff
@@ -0,0 +1,57 @@
+module HammerCLIForeman
+  module SSHKeyCommons
+    def self.included(base)
+      base.option "--user_id", "USER_ID", _("ID of the user to add the SSH Key to")
+    end
+
+    def request_params
+      params = super
+      params["user_id"] = option_user_id
+      params
+    end
+  end
+
+  class SSHKeys < HammerCLIForeman::Command
+    resource :ssh_keys
+    command_name "ssh-keys"
+    desc _("Manage SSH Keys sources.")
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      include SSHKeyCommons
+
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :fingerprint, _("Fingerprint")
+        field :length, _("Length")
+        field :created_at, _("Created at"), Fields::Date
+      end
+
+      build_options
+    end
+
+    class CreateCommand < HammerCLIForeman::CreateCommand
```

Comment:
It would be good to add an option `--key-file` that would take a path to the pub key, read it from there and pass it to the `key` parameter. It should be easy to add with https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/options/normalizers.rb#L134

Reply:
Check.